tests/assets/amazon-eks-vpc.json

```diff
@@ -0,0 +1,1319 @@
+                },
+                "RouteTableId": {
+                    "Ref": "PrivateRouteTable01"
+                }
+            }
+        },
+        "PrivateSubnet02RouteTableAssociation": {
+            "Type": "AWS::EC2::SubnetRouteTableAssociation",
+            "Properties": {
+                "SubnetId": {
+                    "Ref": "PrivateSubnet02"
+                },
+                "RouteTableId": {
+                    "Ref": "PrivateRouteTable02"
+                }
+            }
+        },
+        "PrivateSubnet03RouteTableAssociation": {
+            "Type": "AWS::EC2::SubnetRouteTableAssociation",
+            "Properties": {
+                "SubnetId": {
+                    "Ref": "PrivateSubnet03"
+                },
+                "RouteTableId": {
+                    "Ref": "PrivateRouteTable03"
+                }
+            }
+        },
+        "PrivateSubnet04RouteTableAssociation": {
+            "Type": "AWS::EC2::SubnetRouteTableAssociation",
+            "Properties": {
+                "SubnetId": {
+                    "Ref": "PrivateSubnet04"
+                },
+                "RouteTableId": {
+                    "Ref": "PrivateRouteTable04"
+                }
+            }
+        },
+        "PrivateSubnet05RouteTableAssociation": {
+            "Type": "AWS::EC2::SubnetRouteTableAssociation",
+            "Properties": {
+                "SubnetId": {
+                    "Ref": "PrivateSubnet05"
+                },
+                "RouteTableId": {
+                    "Ref": "PrivateRouteTable01"
+                }
+            }
+        },
+        "PrivateSubnet06RouteTableAssociation": {
+            "Type": "AWS::EC2::SubnetRouteTableAssociation",
+            "Properties": {
+                "SubnetId": {
+                    "Ref": "PrivateSubnet06"
+                },
+                "RouteTableId": {
+                    "Ref": "PrivateRouteTable02"
+                }
+            }
+        },
+        "PrivateSubnet07RouteTableAssociation": {
+            "Type": "AWS::EC2::SubnetRouteTableAssociation",
+            "Properties": {
+                "SubnetId": {
+                    "Ref": "PrivateSubnet07"
+                },
+                "RouteTableId": {
+                    "Ref": "PrivateRouteTable03"
+                }
+            }
+        },
+        "PrivateSubnet08RouteTableAssociation": {
+            "Type": "AWS::EC2::SubnetRouteTableAssociation",
+            "Properties": {
+                "SubnetId": {
+                    "Ref": "PrivateSubnet08"
+                },
+                "RouteTableId": {
+                    "Ref": "PrivateRouteTable04"
+                }
+            }
+        },
+        "ControlPlaneSecurityGroup": {
+            "Type": "AWS::EC2::SecurityGroup",
+            "Properties": {
+                "GroupDescription": "Cluster communication with worker nodes",
+                "VpcId": {
```

Comment:
Do we need to add any rules to this security group?

Reply:
I don't think so.